sdk/personalizer/Azure.AI.Personalizer/src/Customizations/MultiSlotRankRequest.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```

Comment:
We should name the containing folder `Models`.
That way we will have symmetry with `Models` and `Generated\Models` for the partial types.